backend/internal/storage/postgres/schema/media.go

```diff
@@ -228,32 +229,43 @@ func (r *MediaRepository) GetExistingAlbumBySpotifyID(ctx context.Context, id st
 
 func (r *MediaRepository) AddAlbum(ctx context.Context, album *models.Album) (*models.Album, error) {
 	query :=
-		`INSERT INTO album (title, release_date, cover, country, spotify_id)
-		 VALUES ($1, $2, $3, $4, $5)
-		 RETURNING id;`
+		`
+		WITH inserted_album AS (
+		 INSERT INTO album (title, release_date, cover, spotify_id)
+		 VALUES ($1, $2, $3, $4)
+		 ON CONFLICT (spotify_id) DO NOTHING
+		 RETURNING id
+		)
+		 SELECT id FROM inserted_album
```

Comment:
without this extra CTE and select, it was returning nothing for rows that already existed instead of returning the existing id